docs/serving/app-security/security-guard-quickstart.md

```diff
@@ -0,0 +1,174 @@
+# Security-Guard Monitoring Quickstart
+
+Before starting the tutorial, learn [about Security-Guard](./security-guard-about.md)
+
+In this tutorial, deploy a Hello World service and protect it with Guard.
+
+## Before you begin
+
+Using Guard requires that your cluster will use an enhanced queue-proxy image.
+In addition, Guard include automation for auto-learning a per service Guardian. Auto-learning requires you to deploy a guard-service on your kubernetes cluster.
+
+To start this tutorial, after installing Knative Serving, run the following procedure to replace your queue-proxy image and deploy a guard-service in the current namespace.
+
+=== "Install from source"
+
+    1. Clone the security-guard repository using `git clone git@github.com:knative-sandbox/security-guard.git`
+
+    1. Do `cd security-guard`
+
+    1. Run `ko apply -Rf ./config`
+
+=== "Install from released images and yamls"
+
+    Use released images to update your system to enable Guard:
+
+    1. Set the feature named `queueproxy.mount-podinfo` to `allowed` in the config-features ConfigMap.
+
+        An easy way to do that is using:
+
+        ```
+        kubectl apply -f https://raw.githubusercontent.com/knative-sandbox/security-guard/release-0.1/config/deploy/config-features.yaml
+        ```
+
+    1. Set the deployment parameter `queue-sidecar-image` to `gcr.io/knative-releases/knative.dev/security-guard/cmd/queue` in the config-deployment ConfigMap.
+
+        An easy way to do that is using:
+
+        ```
+        kubectl apply -f https://github.com/knative-sandbox/security-guard/releases/download/v0.1.0/queue-proxy.yaml
+        ```
+
+    1. Add the necessary Guard resources to your cluster using:
+
+        ```
+        kubectl apply -f https://raw.githubusercontent.com/knative-sandbox/security-guard/release-0.1/config/resources/gateAccount.yaml
+        kubectl apply -f https://raw.githubusercontent.com/knative-sandbox/security-guard/release-0.1/config/resources/serviceAccount.yaml
+        kubectl apply -f https://raw.githubusercontent.com/knative-sandbox/security-guard/release-0.1/config/resources/guardiansCrd.yaml
+        ```
+
+    1. Deploy `guard-service` on your system to enable automated learning of micro-rules. In the current version, it is recommended to deploy `guard-service` in any namespace where knative services are deployed.
+
+        An easy way to do that is using:
+
+        ```
+        kubectl apply -f https://github.com/knative-sandbox/security-guard/releases/download/v0.1.0/guard-service.yaml
+        ```
```

Comment:
Installing etc here should be steps in the procedure, not part of a "before you begin". Please reformat this to make it part of one procedure, e.g.

1. Install X and Y
2. Set parameters
3. Create resources
4. Deploy stuff
5. Create a service